news/2017-10-11-qat-gazette-october-2017.md

```diff
@@ -0,0 +1,68 @@
+
+## The Upheaval
+
+The Upheaval was probably the biggest event of changes affecting the Quality Assurance Team and the Beatmap Nominators we have seen for a long time. For these who may have missed the details, you can still read more about it in the [Development Forum Thread](https://osu.ppy.sh/community/forums/topics/635507)!
+
+### No More Tiers
+
+As we discussed last month, the biggest point in the Upheaval was **the removal of the Beatmap Nominator Tier system**. As you may know, we previously split the osu! Beatmap Nominators into two groups - Tier 1 and Tier 2. Tier 1 members were able to make the first nomination for maps only, whereas Tier 2 members were able to nominate and also qualify beatmaps. 
+
+Many people were not happy about these changes because of the lack of Tier 2 nominators. The amount of nominated beatmaps increased, while the amount of qualified beatmaps was reduced a lot. With the Upheaval, we have replaced the Tier system with an alternative **“probation” system** (more of this later in the blog!), allowing all Nominators to nominate and qualify beatmaps again! 
+
+### Beatmap Nominator Rewards
+
+Another point of the Upheaval were the Beatmap Nominator rewards! You may have already seen that all Beatmap Nominators received a “Beatmap Nominator” forum title and a special purple forum color. The older players among you will maybe know this color, as it’s really similar to the old Mapping Assistance Team color. It is planned for members of the Beatmap Nominators to also receive further rewards for their effort, including special titles for consistently active members over long periods of time. How we judge the awarding of this **Elite Nominator** title is still being decided on.
+
+### QAT Leaders
+
+The last important point of the Upheaval was the determination of a **Quality Assurance Team Leader**. The QAT Leader is supposed to promote discussion, organise further development, and provide a communication line between game developers and the QAT / Beatmap Nominators. Three of our QATs, [Deif](https://osu.ppy.sh/users/318565), [JBHyperion](https://osu.ppy.sh/users/4879508) and [Okorin](https://osu.ppy.sh/users/1623405) proposed themselves for the position, and after a voting period open to all other QATs and Beatmap Nominators, [Okorin](https://osu.ppy.sh/users/1623405) and [JBHyperion](https://osu.ppy.sh/users/4879508) were elected to the position. They will continue to push forward the remaining proposals from the Upheaval and beyond!
+
+## Beatmap Nominator Applications
+
+**New Beatmap Nominators have arrived once again!** Applications for osu!taiko, osu!catch and osu!mania from September have come to an conclusion, and we would like to welcome following new member to the Beatmap Nominators:
```

Comment:
"we would like to welcome the* following new members* to the Beatmap Nominators"

Reply:
added sssss